sync/class.jetpack-sync-full.php

```diff
@@ -350,8 +423,10 @@ public function set_status_sending_started() {
 		 *
 		 * @since 4.1
 		 */
+
```

Comment:
Should there be an empty newline here? Also, the `@since` should probably be `4.2.0`.